submissions/SofiiaTrokhymchuk/frogger-game/app.js

```diff
@@ -0,0 +1,147 @@
+function createWinLossPointsBlock(){
+    const pointsBlock = document.createElement('div');
+    pointsBlock.style.cssText = `
+        font-family: Helvetica; 
+        font-size: 30px; 
+        width: 50%;
+        margin: 30px auto -10px;
+        display: flex;
+        justify-content: space-evenly;
+    `
+    pointsBlock.innerHTML = `
+        <span>Wins: 0</span>
+        <span>Losses: 0</span>
+    `;
+    document.body.prepend(pointsBlock);
+    return pointsBlock;
+}
+
+function addWinLossPointsToBlock(pointsBlock, player){
+    pointsBlock.innerHTML = `
+        <span>Wins: ${player.wins}</span>
+        <span>Losses: ${player.loses}</span>
+    `;
+}
+
+const pointsBlock = createWinLossPointsBlock();
+
+const TILE_WIDTH = 101;
+const TILE_HEIGHT = 83;
+
+const BOARD_WIDTH = TILE_WIDTH * 4;
+const BOARD_HEIGHT = TILE_HEIGHT * 5;
+
+const CHARACTERS_WIDTH = 101;
+const CHARACHTERS_HEIGHT = 171;
+
+const SIZE_DIFFERENCE = 20;
+
+const INITIAL_ENEMY_X = -CHARACTERS_WIDTH;
+const ENEMY_MIN_SPEED = 100;
+const ENEMY_MAX_SPEED = 500;
+
+const INITAL_PLAYER_X = BOARD_WIDTH / 2;
```

Comment:
As a peer developer I add yet another column to the board. Is there a more reliable way to define this so that player is still aligned against tiles?